cypress_test/integration_tests/desktop/writer/searchbar_spec.js

```diff
@@ -142,4 +142,20 @@ describe(['tagdesktop', 'tagnextcloud', 'tagproxy'], 'Searching via search bar'
 		desktopHelper.assertScrollbarPosition('vertical', 200, 305);
 		desktopHelper.assertVisiblePage(3, 4, 6);
 	});
+
+	it.only('Search input should keep the focus after a part change', function() {
```

Comment:
forgotten `only`

Reply:
Ah, thank you. Do we still need this commit? I think you have another fix?

Reply:
I added your test to my fix. I think better to not check the hardcoded "id". But would be good if you look if  my patch is ok.